lib/tasks/exotel_tasks.rake

```
@@ -0,0 +1,30 @@
+namespace :exotel_tasks do
+  desc 'Whitelist patient phone numbers on Exotel'
+  task whitelist_patient_phone_numbers: :environment do
+    require 'exotel_tasks/whitelist_phone_numbers'
+
+    data_file = ENV['JSON_FILE_PATH']
```

Let's use `ENV.fetch` here so we can fail early if the params are missing.